test/parallel/test-process-redirect-warnings-env.js

```diff
@@ -6,14 +6,15 @@
 // opened and the contents are validated
 
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 const fs = require('fs');
 const fork = require('child_process').fork;
 const path = require('path');
 const assert = require('assert');
 
 common.refreshTmpDir();
 
-const warnmod = require.resolve(`${common.fixturesDir}/warnings.js`);
+const warnmod = require.resolve(`${fixtures.fixturesDir}/warnings.js`);
```

Comment:
This should probably be replaced with `fixtures.path("/warnings.js")`

Reply:
@BridgeAR Indeed. Thanks